libs/blocks/text/text.js

```diff
@@ -13,17 +26,31 @@ export default function init(el) {
   el.classList.add('text-block', 'con-block');
   let rows = el.querySelectorAll(':scope > div');
   if (rows.length > 1) {
-    el.classList.add('has-bg');
+    if (rows[0].textContent !== '') el.classList.add('has-bg');
     const [head, ...tail] = rows;
     decorateBlockBg(el, head);
     rows = tail;
   }
+  const helperClasses = [];
+  let blockType = 'text';
   const size = getBlockSize(el);
-  decorateBlockText(el, blockTypeSizes.text[size]);
+  const longFormVariants = ['inset', 'long-form', 'bio'];
+  longFormVariants.forEach((v, i) => {
```

Comment:
Nit: one letter parameters are a little difficult to understand.

Reply:
Agree, I updated this to use better function var names.